README.md

```diff
@@ -32,10 +33,18 @@ $ forge snapshot
 ```
 
 ### Test 
+For validating that tests are working run 
+
+```shell 
+$ forge test --fork-url "https://rpc.gnosis.gateway.fm" -vvvv --fuzz-runs 1
+```
+
+For a full run of tests run the following command, note that it may take a significant amount of time 
```

Comment:
Run on sentence. Split into two. Missing period. Maybe give a rough estimate of how long is a "significant amount of time".

Reply:
addressed with a036006